sdk/core/Azure.Core/src/AzureExtentions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure
+{
+    public static class AzureExtensions
```

Comment:
Should we add a comment that nothing gets added here without explicit approval from the Arch Board _(or the .NET part of the Arch Board)_?

Reply:
Isn't this the case for all Track 2 APIs?

Reply:
It _should_ be the case, yes.  I suppose stuff won't be able to accidentally sneak in once we have automated compat analysis running on PRs anyway.